server/config/forms/changePrisonForm.js

```diff
@@ -0,0 +1,12 @@
+const { joi, validations } = require('./validations')
+const { buildValidationSpec } = require('../../services/validation')
+
+const { requiredStringMsg } = validations
+
+const transientSchema = joi.object({
+  agencyId: requiredStringMsg('What prison did the use of force take place in?'),
```

Comment:
I think we talked about this but wouldn't bother to handle the extra complexity of using the form processor validation stuff. It's only one field. 

You might has well just check it in the route.

Reply:
done. 
Validation removed from form processor and done directly in routes/location.js instead.
I do remember the conversation but thought it was primarily about whether we should have validation at all. When it was decided to keep the validation, I then assumed that  it should be done like every other input in the app, i.e via Joi.